source/guide_synapse.rst

```diff
@@ -45,6 +45,8 @@ You need a running :lab:`Postgresql <guide_postgresql>` database server, a dedic
   Enter it again:
   [isabell@stardust ~]$ createdb \
     --encoding=UTF8 \
+    ----lc-collate=C \
+    ----lc-ctype=C \
```

Comment:
```suggestion
    --lc-collate=C \
    --lc-ctype=C \
```

The `--help` shows these options with just two dashes (`--`), do we really need four of them?